wagtailmedia/tests/testapp/migrations/0002_auto_20210729_0344.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 3.1.13 on 2021-07-29 08:44
```

Comment:
nitpick: I think using named migrations everywhere makes it much easier to understand the changes

Reply:
done